apps/docs/app/examples/navbar/disabled-blur/page.tsx

```diff
@@ -1,6 +1,16 @@
 "use client";
 
-import {Navbar, NavbarBrand, NavbarContent, NavbarItem, Link, Button} from "@nextui-org/react";
+import {
+  Navbar,
+  NavbarBrand,
+  NavbarContent,
+  NavbarItem,
+  Link as NextUiLink,
+  LinkProps,
+  Button,
+} from "@nextui-org/react";
+
+const Link = (props: LinkProps) => <NextUiLink {...props} onClick={(e) => e.preventDefault()} />;
```

Comment:
Please implement the `Link` directly from `@nextui-org/react` since preventing the event is only useful for the docs but might be confusing when someone using this implementation

Reply:
Not sure how to do that?